packages/aws-cdk-lib/aws-cloudfront/lib/origin.ts

```diff
@@ -2,6 +2,21 @@ import { Construct } from 'constructs';
 import { CfnDistribution } from './cloudfront.generated';
 import { Duration, Token } from '../../core';
 
+/**
+ * The selection criteria for the origin group.
+ */
+export enum OriginSelectionCriteria {
+  /**
+   * Default selection behavior.
+   */
+  DEFAULT='default',
+
+  /**
+   * Selection based on media quality.
```

Comment:
can we add a note here that it is only a valid option for AWS Elemental MediaPackage v2 origins?

Reply:
added this